Support/Multitenancy/Multiple-Ironic-conductors/install-ironic.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -eux
```

Comment:
```suggestion
set -eu
```
The only one where we handle (fake, but still) usernames/passwords, we set `-x` for spamming them out.. Hilarious :)